cluster/gce/gci/health-monitor.sh

```diff
@@ -39,7 +39,7 @@ function container_runtime_monitoring {
   fi
   # Container runtime startup takes time. Make initial attempts before starting
   # killing the container runtime.
-  until timeout 60 ${healthcheck_command} > /dev/null; do
+  until timeout 60 "${healthcheck_command}" > /dev/null; do
```

Comment:
Look at the definition of healthcheck_command, it has multiple items

Reply:
received

Reply:
still looks like this is running `docker ps pods` rather than `docker` `ps` `pods`?

Reply:
@fejta where do you see multiple items? It looks like it's supposed to run either `docker ps` or `${KUBE_HOME}/bin/crictl pods` based on the value of `CONTAINER_RUNTIME`

Reply:
Right, and both of those are multiple items `"docker ps"` != `"docker" "ps"`, making the quoting here inappropriate.

Reply:
Yeah, this should be an array then, and use `"${healthcheck_command[@]}"` to call it.

@xichengliudui can you convert `healthcheck_command` to an array? It's set on lines 36 and 38.

Reply:
this still needs addressing